usr/share/rear/output/ISO/Linux-i386/850_check_for_errors.sh

```diff
@@ -1,2 +1,4 @@
 [[ $DEBUGSCRIPTS -eq 1 ]] && return   # to avoid throwing an error in DEBUG mode
```

Comment:
By the way:
I have no idea why we "avoid throwing an error in DEBUG mode"?
It surprises me that ReaR behaves differebnt in debug mode.
In this case it means an issue does not appear in debug mode.
Why is that done?
It me it looks as it that defeats the purpose of the debug mode.

Reply:
@jsmeix 

I think I understand it - grepping for messages in DEBUGSCRIPTS mode is dangerous, the log contains more than messages - it contains every command executed. So, if there is a command like `grep -iq "no space left"`, the log will contain something like
```
+++ grep -iq "no space left"
```
and the grep will succeed. So, disabling it in DEBUGSCRIPTS makes sense.

That said, I am afraid that this grep check does not work anymore, since stderr from external commands (and that's what we are looking for, I assume - messages from external commands, not messages from ReaR itself) is redirected to a separate file and does not appear in the main log anymore. If we want to keep this check, we should grep the file where stderr gets redirected (and we could also drop the DEBUGSCRIPTS test).

Reply:
Oh - now I see it!

The test
```
gep -iq "no space left" $RUNTIME_LOGFILE && Error "write error: No space left on device"
```
is wrong in any case because regardless in what mode ReaR runs
"no space left" can unpredictably appear at arbitrary places
in RUNTIME_LOGFILE.

This test in a separated script cannot work so the whole
usr/share/rear/output/ISO/Linux-i386/850_check_for_errors.sh
should be removed and a proper test should be implemented
at the specific code place where the command runs that
may fill up all space on the device.